### DISCUSSION TITLE
Gas Turbine + HRSG Example

### DISCUSSION THREAD

**Discussion by energydatachpguy** [oemof/tespy#589] [Q&A] [answered]
Is there any update on when the example for the GT + HRSG will be available on the advanced tutorials page? Separately, but equally useful, would be an example that shows a full power plant cycle with boiler , backpressure turbine, steam export (with condensate return variable of <1), along with cooling tower and a DA. It is possible different pieces of examples like this exist in the tutorial page that I didn't come across yet. I would like to use the library to assist in the CHP cycle that will sit ahead of a carbon capture plant, one plant design would involve boilers + stg other would involve gt + hrsg. Thanks so much.

**Comment by fwitte** [comment]
Three time's the charm :)!

> Is there any update on when the example for the GT + HRSG will be available on the advanced tutorials page?

So this is the current state of the new tutorial (it is only drafted) https://github.com/oemof/tespy/pull/558, https://tespy--558.org.readthedocs.build/en/558/tutorials/hrsg_process_steam.html#tespy-tutorial-hrsg-label. It requires quite a bit of refining at the current state. Particularly, there was an issue in the mass flow ratios (flue gas vs. steam) because there was a temperature setting, that did not make much sense at connection 15 or 16. That has been found, now it is a question of finding the time to write up a nice story line. Once I find the time to complete the tutorial, I will update you. Timeframe for that would be somewhen in Q1 of 2025.

In the meantime, there is the SEGS example, which features one of the solar energy generating systems from the early 90s in the US (iirc) https://github.com/fwitte/segs_exergy. It is not CHP but also features an interesting topology.

> Separately, but equally useful, would be an example that shows a full power plant cycle with boiler , backpressure turbine, steam export (with condensate return variable of <1), along with cooling tower and a DA. It is possible different pieces of examples like this exist in the tutorial page that I didn't come across yet.

Agree, I would invite you to join one of the next user meetings (or we can have a bilateral meeting) and to discuss possible ideas, thank you!

Best

Francesco

**Reply by PabloMBarral** [reply, marked as answer]
Hi! We are working. The example should be finished by march. Parameter definition was tricky, because the hrsg wasn't that simple. 

I've sent you the draft that is, while still not neat, almost ready. 

I'll keep you informed, and I will try to contribute to the tespy doc with that point.

**Reply by energydatachpguy** [reply]
Hey Pablo and Francisco,

Curious if there is an update on a simplified Gas Turbine + HRSG style example.

This would be very valuable for some of the calculations we are trying to do at our firm. 

We are looking at a small gas turbine (10 MW) , with only an economizer, and drum (with blowdown included), with the drum at 170psig creating saturated steam. 

Ideally we would like the option to duct fire and add additional heat to toggle the steam output. 

It would be helpful for us to run at different ambient temperatures or electric load requirements. 

Thank you in advance.

**Reply by PabloMBarral** [reply]
I'll prepare something! Give me this week.

**Reply by fwitte** [reply]
Hi both, thank you for the updates. You see @PabloMBarral, that I have not yet found the time to integrate the tutorial in a descriptive way in the docs. It should find its way there soon, however I have too many other pressing topics right now. I had already started to create the framework for the tutorial here #558.

Also, the tutorial/model requested by you, @energydatachpguy, could be a more simple one, which would also be a nice to have for the documentation. So if any of you is able to provide code + some step by step (bulletpoints maybe) instructions, that would be super helpful to add to the docs :).

Have a nice week!

**Reply by PabloMBarral** [reply]
Only an evap, a heater for supp firing, an economizer, and the gt. Should be straightforward.

The only thing to care is how to determine pinch location

I'll try to write something.
